docs/mkdocs.yml

```diff
@@ -58,6 +58,7 @@ extra_css:
   - stylesheets/splide.min.css
   - stylesheets/extra.css
   - stylesheets/neoteroi-mkdocs.css
+  - https://cdn.jsdelivr.net/npm/@tabler/icons-webfont@3.31.0/dist/tabler-icons.min.css
```

Comment:
Does this mean docs can not be build offline?

Reply:
You can build offline, but there will be some elements fetched via CDN when viewing the docs. That is already the case with some elements  (we fetch bootstrap via CDN for example). Do you think we should target fully offline docs builds?

Reply:
Fully offline docs would be good, at least from releases. That makes full reproducibility easier

Reply:
To be clear, the docs will run "offline" - your browser will just fetch some icons from a CDN as required.